### PR TITLE
Login is working

### PR DESCRIPTION
**Pull request by serenabertozzi**

### CODE REVIEW

**Review by bernardleanse — APPROVED** [review]

**Review by Kntanos — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -5,28 +5,29 @@ import { HomePage } from "./components/HomePage";
 import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
 import { Scoreboard } from "./components/gameState/Scoreboard";
 import { Layout } from "./components/Layout";
-import { Title } from "./components/Title";
+import { Navbar } from "./components/Navbar";
 import { useState } from "react";
-import React from 'react';
+import React from "react";
```
Comment (AJOsmaston) [inline src/App.js:10]: i hate prettier

File: src/components/HomePage.js [hunk 2]
```diff
@@ -4,11 +4,9 @@ import { useState } from "react";
 import Button from "@material-ui/core/Button";
 import { Grid } from "@material-ui/core";
 import { Title } from "./Title";
-import React from 'react';
+import React from "react";
```
Comment (AJOsmaston) [inline src/components/HomePage.js:7]: and this

File: src/components/Title.js [hunk 3]
```diff
@@ -1,33 +1,27 @@
 import { Grid } from "@material-ui/core";
-import React from 'react';
+import React from "react";
```
Comment (AJOsmaston) [inline src/components/Title.js:2]: garbage

File: src/components/gameState/GameView.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React from "react";
```
Comment (AJOsmaston) [inline src/components/gameState/GameView.js:1]: more of this

**Review by AJOsmaston — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (serenabertozzi) [reply src/App.js:10]: <3

**Pull request merged**